src/main/java/uk/gov/pay/adminusers/app/config/AdminUsersModule.java

```diff
@@ -98,7 +98,6 @@ private JpaPersistModule jpaModule(AdminUsersConfig configuration) {
     @Provides
     private NotificationService provideUserNotificationService() {
         return new NotificationService(
-                environment.lifecycle().executorService("2fa-sms-%d").build(),
```

Comment:
Presumably we don’t need this at all because email sending is not async any more?

Reply:
Yeah, none of the calls made by `NotificationService` should be async.